app/src/main/res/values-pl/strings.xml

```diff
@@ -12,60 +12,61 @@
     <string name="ok">Ok</string>
     <string name="failed_to_open_selected_file">Nie udało się otworzyć wybranego pliku</string>
     <string name="permissions_needed">Wymagane są dodatkowe uprawnienia</string>
-    <string name="cell_tower_icon_description">Ikona BTS</string>
+    <string name="cell_tower_icon_description">Ikona wieży komórkowej</string>
     <string name="reports_tab_title">Raporty</string>
     <string name="send_reports">Wyślij raporty</string>
-    <string name="reports_last_uploaded">Ostatnio wysłane raporty: %s</string>
+    <string name="reports_last_uploaded">Raporty były wysłane ostatnio: %s</string>
     <string name="start_scanning">Rozpocznij skanowanie</string>
     <string name="delete_report">Usuń raport</string>
-    <string name="delete_report_confirmation">Czy jesteś pewny że chcesz usunąć ten raport?</string>
+    <string name="delete_report_confirmation">Czy jesteś pewny, że chcesz usunąć ten raport?</string>
     <plurals name="toast_reports_uploaded">
         <item quantity="one">Wysłano %d raport</item>
         <item quantity="few">Wysłano %d raporty</item>
         <item quantity="many">Wysłano %d raportów</item>
-        <item quantity="other">Wysłano %d raportów</item>
     </plurals>
     <string name="notification_sending_reports">Wysyłanie raportów</string>
-    <string name="notification_wireless_scanning_title">Skanowanie bezprzewodowo aktywne</string>
+    <string name="notification_wireless_scanning_title">Skanowanie aktywne</string>
     <plurals name="notification_wireless_scanning_content_reports_created">
         <item quantity="one">Stworzono %d raport</item>
         <item quantity="few">Stworzono %d raporty</item>
         <item quantity="many">Stworzono %d raportów</item>
-        <item quantity="other">Stworzono %d raportów</item>
     </plurals>
-    <string name="notification_wireless_scanning_content_satellite_stats">%1$d/%2$d satelit w użyciu</string>
+    <plurals name="notification_wireless_scanning_content_satellite_stats">
+        <item quantity="one">%1$d⁄%2$d satelity w użyciu</item>
+        <item quantity="few">%1$d⁄%2$d satelitów w użyciu</item>
+        <item quantity="many">%1$d⁄%2$d satelitów w użyciu</item>
+    </plurals>
```

Comment:
You can't change the resource type from string to plural, because resources are accessed by their type (i.e. we would have to do the same change for all languages and then change the code to fetch plurals instead of strings)

I'm not sure if this needs plural forms at all, because the value is a fraction?